src/checkers/anityachecker.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import urllib.request
 import urllib.parse
-from string import Template
+from distutils.version import LooseVersion
```

Comment:
Probably unused import?

Reply:
LooseVersion is not accessed directly from this module, you don't need to import it here.